client/src/www/app/outline_server_repository/index.ts

```diff
@@ -242,17 +242,14 @@ export class OutlineServerRepository implements ServerRepository {
         error.message || 'failed to parse access key'
       );
     }
-    if (config.host.isIPv6) {
```

Comment:
FYI, I removed this. It should be up to the Go code to tell what is supported, otherwise we can't do advanced strategies.